### ISSUE TITLE
Clarification about Labels

### ISSUE THREAD

**Issue by GeorgeBatch** [ncoudray/DeepPATH#108]
Dear authors,

Thank you very much for releasing and documenting your code! 

I read your work and I have a question about the labels you used for training/fine-tuning the inception v3 network. Let's take a 3-way classification (normal vs LUSC vs LUAD). Did you use the same WSI label as the label for every tile in this WSI or did you ask the doctors to manually select big regions on the WSIs with Normal/LUSC/LUAD tissue to then tile and use for training/fine-tuning? The paper mentions "manual annotation" a couple of times, but I do not understand if they were WSI-level annotations (e.g. to compare pathologists' performance with TCGA-labels which you took as gold standard labels) or region-based manual annotations.

Many thanks,
George Batchkala

**Comment by ncoudray** [comment]
Hi George,

 Yes, in this project, the whole slide label was used and assumed for each tile.
 Except the NYU slides where regions were manually labelled, for the Kappa, but "manual annotations", we meant they visually looked at the slides and assigned a label to it.

Best,
Nicolas

**Comment by GeorgeBatch** [comment]
Hi Nicolas,

Thank you very much for the clarification!

Best wishes,
George
